### PR TITLE
Limit SVG width on mobile [Fixes #38]

### LINKED ISSUE #38: Downloads header overflowing
Screen Shot 2022-11-17 at 7 27 33 PM

Downloads header overflowing on mobile view

### PR DESCRIPTION
**Pull request by wackerow**
## Description
- Adds a base case (mobile) of 100% width for GopherDownloads SVG component on /downloads page
- Removes unneeded padding on container.

## Screenshot
@ (320px wide)
image

## Related issue
- [Fixes #38]

### CODE REVIEW

**Comment by nhsz** [comment]
Padding is still required for tablet/desktop (`md` and higher)

Screen Shot 2022-11-21 at 22 13 27

File: src/components/UI/downloads/DownloadsHero.tsx [hunk 1]
```diff
@@ -67,14 +67,12 @@ export const DownloadsHero: FC<DownloadsHero> = ({
           </Stack>
 
           <Stack
-            py={{ base: 0 }}
```
Comment (nhsz) [inline src/components/UI/downloads/DownloadsHero.tsx:70]: Padding is still required for [tablet/desktop ](https://www.figma.com/file/ekzIgwyeVKLtFSAcnA0Q0D/geth-website?node-id=1043%3A8879&t=rol3tipDhhgzwHrC-0)(`md` and higher). Seems like [padding for the content in mobile](https://www.figma.com/file/ekzIgwyeVKLtFSAcnA0Q0D/geth-website?node-id=1077%3A9400&t=rol3tipDhhgzwHrC-0) should be updated too 

Screen Shot 2022-11-21 at 22 13 27

**Review by nhsz — CHANGES_REQUESTED** [review]
Thanks @wackerow, seems we need some minor adjustments related to the padding and it'll be ready to go 😎

**Review by corwintines — APPROVED** [review]
Pushed something to clean this up (my own code from before, just noticed it), but added some padding from the change request

**Review by nhsz — APPROVED** [review]

**Pull request merged**